docs/package.json

```diff
@@ -39,10 +39,9 @@
 		"start": "hugo server"
 	},
 	"dependencies": {
-		"@fluid-tools/api-markdown-documenter": "^0.9.0",
+		"@fluid-tools/api-markdown-documenter": "^0.10.1",
```

Comment:
I would revert this so you can pick up the correct changes from `next` when you next merge it into this branch.